common/app/dev/DevAssetsController.scala

```diff
@@ -14,7 +14,13 @@ object DevAssetsController extends Controller with ExecutionContexts {
       if (MimeTypes.isText(mime)) s"${mime}; charset=utf-8" else mime
     }
 
-    val resolved = new File(s"static/hash/$path").toURI.toURL
+    val hashFile = new File(s"static/hash/$path")
+
+    val resolved = if (hashFile.exists()) {
+      hashFile.toURI.toURL
+    } else {
+      new File(s"static/src/$path").toURI.toURL
+    }
```

Comment:
With this we will lose the benefit of getting a compilation error if a file doesn't actually exist. I.e. it will just result in a 404 instead. I assume we are happy with this?